CONTRIBUTING.md

```diff
@@ -100,6 +100,7 @@ Language Notes:
 1. Use commit messages that follow the conventions here whenever possible:
    http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 1. Use [semantic versioning](http://semver.org/) for tags
+    - Release notes should include summary of new features and breaking changes
```

Comment:
I'm confused because the lines in this commit appear to have been added rather than removed?

Reply:
Oh oops. I must have goofed this up when I was amending commits to try to split this change off into a separate commit :man_facepalming: